src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -160,6 +162,7 @@ const ContributionsPage = () => {
     useMutation(createContributionMutation, {
       onSuccess: newContribution => {
         refetchContributions();
+        queryClient.invalidateQueries({ queryKey: ['allocate-contributions'] });
```

Comment:
can we make the queryKey into a constant and use that for extra type safety.

Reply:
+1

Reply:
Good idea, refactored with https://github.com/coordinape/coordinape/pull/1622/commits/5e990d20aa7115e94ddcbf6bb518499037469cfb